HW4_Sivtsev/protein_tools.py

```diff
@@ -0,0 +1,160 @@
+aminoacid_alphabet_1to3 = {'A': 'Ala', 'R': 'Arg', 'N': 'Asn', 'D': 'Asp', 'C': 'Cys', 
+							'Q': 'Gln', 'E': 'Glu', 'G': 'Gly', 'H': 'His', 'I': 'Ile', 
+							'L': 'Leu', 'K': 'Lys', 'M': 'Met', 'F': 'Phe', 'P': 'Pro', 
+							'S': 'Ser', 'T': 'Thr', 'W': 'Trp', 'Y': 'Tyr', 'V': 'Val'}
+                            
+molecular_mass = {'A': 89.094, 'R': 174.203, 'N': 132.119, 'D': 133.104, 'C': 121.154, 
+                  'E': 147.131, 'Q': 146.146, 'G': 75.067, 'H': 155.156, 'I': 131.175,
+                  'L': 131.175, 'K': 146.189, 'M': 149.208, 'F': 165.192, 'P': 115.132,
+                  'S': 105.093, 'T': 119.119, 'W': 204.228, 'Y': 181.191, 'V': 117.148}
+
+
+def is_prot(prot: str) -> bool:
+    """
+    Checks is given sequence a protein
+    Arguments:
+        prot (str) - aminoacid sequence of protein
+    Return:
+        bool if sequence is correct
+        ValueError('Please  check proteins sequences') if there were wrong symbols
+    """
+    aas = {'A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y'}
+    prot = prot.upper()
+    uniq_aas = set(prot)
+    aa_test = (uniq_aas <= aas)
+    if aa_test == 0:
+        raise ValueError('Please  check proteins sequences')
+    return True
```

Comment:
Лучше использовать функционал множеств, и результат операции не записывать в отдельную переменную
```suggestion
    aa_test = (uniq_aas <= aas)
    if uniq_aas.issubset(aas):
        return True
    else:
        raise ValueError('Please  check proteins sequences')
```